packages/matrix/tests/room-creation.spec.ts

```diff
@@ -27,7 +27,8 @@ import {
 test.describe('Room creation', () => {
   let synapse: SynapseInstance;
 
-  test.beforeEach(async ({ page }) => {
+  test.beforeEach(async ({ page }, testInfo) => {
+    testInfo.setTimeout(testInfo.timeout + 30000);
```

Comment:
i think there is actually a special API for playwrite that you specify that this is a slow test and more timeout will be given to it. https://playwright.dev/docs/api/class-test#test-slow